api/bus.go

```diff
@@ -35,10 +35,11 @@ const (
 
 	DefaultBucketName = "default"
 
-	SettingContractSet   = "contractset"
-	SettingGouging       = "gouging"
-	SettingRedundancy    = "redundancy"
-	SettingUploadPacking = "uploadpacking"
+	SettingContractSet      = "contractset"
+	SettingGouging          = "gouging"
+	SettingRedundancy       = "redundancy"
+	SettingS3Authentication = "s3authentication"
```

Comment:
hm name is not great, maybe `authentication` and then do 
```
{
    S3:{
        V4Keypairs?
    }
}
```

Reply:
I don't mind a different name but I'm not a huge fan of changing the setting itself for the sake of a different name.